content/pages/de/meetups.rst

```diff
@@ -14,19 +14,19 @@ Meetups
         :width: 400px
 
 Meetups
-~~~~~~~
+~~~~~~~~
 
-We have monthly meetups with talks and workshop on various python related and development in general and data topics.
+In unserem monatlichen Meetup kannst du dich auf Workshops und Vorträge zu verschiedenen Python Themen z.B. aus dem Bereich Softwareentwicklung oder Data Science freuen.  
+Außerdem beschäftigen wir uns in unseren Setup-Events mit grundlegenden Themen, die dir deinen Start in die Programmierung mit Python erleichtern.
+Unsere aktuellen Events findest du hier: `PyLadies Berlin Meetup group <https://www.meetup.com/PyLadies-Berlin/>`_.
 
-You can find and join upcoming events on our `Meetup group <https://www.meetup.com/PyLadies-Berlin/>`_.
+Bei unseren Events findest du Vorträge zu sehr speziellen Themen, aber auch Einsteiger-Workshops. 
+Unsere Events beginnen wir oft mit einem 10min. "Non-coding superpower" Vortrag, da es viele Themen gibt, die den Tech-Alltag bereichern und nichts mit Programmierung zu tun haben.
 
-We do have varying levels in different meetups from novice to expert in specific topics. We always try to
-include a "Non-coding superpower" 10 mins talk as tech life and life is much more than coding!
-
-Speak in our meetup
+Erzähle uns was!
 ~~~~~~~~~~~~~~~~~~~
 
-We love to hear what you have to share with the community, from beginners topic to a project you worked on
-or a hands on session or a non-coding superpower!
+Wir freuen uns immer über neue interessanten Themen, die du mit der Python Community teilen möchtest. 
+Angefangen von einer kurzen 10min. Präsentation einer bestimmten Frameworks bis hin zu einem ganztägigen Workshop oder einer Präsentation deines ganz eigenen Python Projektes.
 
-Reach out to us via email or slack.
\ No newline at end of file
+Kontaktiere uns einfach via E-Mail oder Slack.
```

Comment:
you can add a link to the contact page here like this

`E-mail or Slack </contact.html>`_

Same for the english page.